zebra/zebra_vty.c

```diff
@@ -751,20 +791,27 @@ DEFPY(ip_route_blackhole_vrf,
       "Set tag for this route\n"
       "Tag value\n"
       "Distance value for this route\n"
-      MPLS_LABEL_HELPSTR)
+      MPLS_LABEL_HELPSTR
+      "Table to configure\n"
+      "The table number to configure\n")
 {
 	VTY_DECLVAR_CONTEXT(vrf, vrf);
 	struct zebra_vrf *zvrf = vrf->info;
 
+	if (table_str && !vrf_is_backend_netns()) {
+		vty_out(vty,
+			"%% table param only available when running on netns-based vrfs\n");
+		return CMD_WARNING_CONFIG_FAILED;
+	}
 	/*
 	 * Coverity is complaining that prefix could
 	 * be dereferenced, but we know that prefix will
 	 * valid.  Add an assert to make it happy
 	 */
 	assert(prefix);
 	return zebra_static_route_leak(vty, zvrf, zvrf, AFI_IP, SAFI_UNICAST,
-				       no, prefix, mask_str, NULL, NULL, NULL,
-				       flag, tag_str, distance_str, label);
+			       no, prefix, mask_str, NULL, NULL, NULL,
+			       flag, tag_str, distance_str, label, table_str);
```

Comment:
surprised checkpatch didn't complain about this, but it would be good to change it to add the new argument without changing the previous line

Reply:
ok. good idea.